i18n/en.json

```diff
@@ -3672,6 +3672,13 @@
   "last_users_message.others": "{numOthers} others ",
   "last_users_message.removed_from_channel.type": "were **removed from the channel**.",
   "last_users_message.removed_from_team.type": "were **removed from the team**.",
+  "learn_more_about_trial.modal.ldapDescription": "Use AD/LDAP groups to organize and apply actions to multiple users at once. Manage team and channel memberships, persmissions and more.",
+  "learn_more_about_trial.modal.ldapTitle": "Synchronize your Active Directory/LDAP groups",
+  "learn_more_about_trial.modal.systemConsoleDescription": "Use System Roles to give designated users read and/or write access to select sections of System Console.",
+  "learn_more_about_trial.modal.systemConsoleTitle": "Provide controlled access to the System Console",
+  "learn_more_about_trial.modal.useSsoDescription": "Sign on quickly and easily with our SSO feature that works with OpenID, SAML, Google, and 0365.",
+  "learn_more_about_trial.modal.useSsoTitle": "Use SSO (with OpenID, SAML, Google, 0365)",
+  "learn_more_trial_modal_step.learnMoreAboutFeature": "Learn more about this feature.",
   "learn_more_trial_modal.pretitle": "With Enterprise, you can....",
```

Comment:
Four dots. Maybe better to use the ellipsis char `…`

Reply:
nice catch!